### PR TITLE
documentation: update contributing guide

### PR DESCRIPTION
**Pull request by PascalRepond**
Co-Authored-by: Pascal Repond <pascal.repond@rero.ch>

### CODE REVIEW

**Review by rerowep — APPROVED** [review]

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -76,71 +76,68 @@ If you are proposing a feature:
...
+Pull Request Guidelines
+-----------------------
 
-      $ ./run-tests.sh
+Before you submit a pull request, check that it meets these guidelines:
 
-   The tests will provide you with test coverage and also check PEP8
-   (code style), PEP257 (documentation), flake8 as well as build the Sphinx
-   documentation and run doctests.
+1. The pull request should include tests and must not decrease test coverage.
```
Comment (jma) [inline CONTRIBUTING.rst:89]: `must` -> `should`. Satisfying this condition is sometime not possible.

File: CONTRIBUTING.rst [hunk 2]
```diff
@@ -76,71 +76,68 @@ If you are proposing a feature:
...
+2. If the pull request adds functionality, the docs should be updated. Put
```
Comment (jma) [inline CONTRIBUTING.rst:90]: I do not understand this point sorry.

File: CONTRIBUTING.rst [hunk 3]
```diff
@@ -76,71 +76,68 @@ If you are proposing a feature:
...
+with fellow developers, so be clear and brief.
 
-        # globally
-        $ git config --global commit.template /path/.gitmessage.txt
+(Inspired by `How to Write a Git Commit Message
+<https://chris.beams.io/posts/git-commit/>`_)
 
-        # locally
-        $ git config --local commit.template .gitmessage.txt
+1. `Separate subject from body with a blank line
```
Comment (jma) [inline CONTRIBUTING.rst:103]: Please put a `.` after each item in the list.

File: CONTRIBUTING.rst [hunk 4]
```diff
@@ -76,71 +76,68 @@ If you are proposing a feature:
...
+      Are there side effects or other unintuitive consequences of this
+      change? Here's the place to explain them.
 
-8. Submit a pull request through the GitHub website.
+    * Use words like "Adds", "Fixes" or "Breaks" in the listed bullets to help
+      others understand what you did.
 
-Pull Request Guidelines
------------------------
+    * If your commit closes or addresses an issue, you can mention
```
Comment (jma) [inline CONTRIBUTING.rst:131]: We can also document how to refer to an issue of an other project.

File: CONTRIBUTING.rst [hunk 5]
```diff
@@ -76,71 +76,68 @@ If you are proposing a feature:
...
+   component: summarize changes in 50 char or less
 
-   .. code-block:: console
+    * More detailed explanatory text, if necessary. Formatted using
+      bullet points, preferably `*`. Wrapped to 72 characters.
 
-      $ git add .
-      $ git commit -s
-      $ git push origin name-of-your-bugfix-or-feature
+    * Explain the problem that this commit is solving. Focus on why you
```
Comment (jma) [inline CONTRIBUTING.rst:123]: We can also to explain how to specify a data migration procedure.

**Review by jma — CHANGES_REQUESTED** [review]

**Review by BadrAly — APPROVED** [review]

**Review by jma — APPROVED** [review]

**Pull request merged**